integration-tests/src/test/scala/eu/neverblink/jelly/integration_tests/rdf/TestStream.scala

```diff
@@ -23,3 +23,7 @@ trait TestStream:
   def quadSink(os: OutputStream)(using ExecutionContext): Sink[RdfStreamFrame, Future[Done]]
 
   def graphSink(os: OutputStream)(using ExecutionContext): Sink[RdfStreamFrame, Future[Done]]
+
+  def supportsRdfStar: Boolean = true
+  
+  def supportsRdf12: Boolean = false
```

Comment:
Please add a note here on why this is needed... especially because Jelly does not really support RDF 1.2.